.gitattributes

```
@@ -0,0 +1,14 @@
+* text=auto
+
+*.java          text
+
+*.ftl           eol=lf
+*.ics           eol=lf
+*.json          eol=lf
+*.mustache      eol=lf
+*.properties    eol=lf
+*.tm            eol=lf
+*.txt           eol=lf
+*.vm            eol=lf
+*.xml           eol=lf
+
```

I wonder whether we want to list all suffixes for lf? Wouldn't it be more practical to say lf for all and specify only the few that are known to require Windows EOLs? Something like  
```suggestion
* text=auto eol=lf
*.cmd    text eol=crlf
```

---

Yeah, probably. Let me try that config.